packages/commonjs/test/fixtures/.eslintrc

```diff
@@ -10,6 +10,8 @@
     "import/prefer-default-export": "off",
     "import/extensions": "off",
     "import/no-unresolved": "off",
-    "@typescript-eslint/no-unused-vars": "off"
+    "@typescript-eslint/no-unused-vars": "off",
+    "camelcase": "off",
+    "no-underscore-dangle": "off"
```

Comment:
Had to add some eslint ignore, what is the prefered approach here?

Reply:
> Had to add some eslint ignore, what is the prefered approach here?

If I understand correctly, you did this due to the `const` at the top.
It's pretty standard for `const`s to be upper-underscore-case, and it looks like eslint supports that out of the box:
>  If ESLint decides that the variable is a constant (all uppercase), then no warning will be thrown  

(https://eslint.org/docs/rules/camelcase)

So it's weird you had to do this.
Anyway I'm for using a local `eslint-disable-line` on the same line. So other instances will still warn like it usually does, to keep the style aligned.

Reply:
The lintings errors come from generated output in the test fixtures, I can't make those const or add linting comments unfortunately.

Reply:
Oh but you can exclude the tests folder, or override settings for the tests folder